resource_customizations/core.spinkube.dev/SpinApp/health.lua

```diff
@@ -0,0 +1,56 @@
+hs = {}
+
+-- Check if status exists
+if not obj.status then
+  hs.status = "Degraded"
```

Comment:
Maybe progressing instead? Presumably waiting on the controller to do its thing.

Reply:
fixed - thanks!